assets/src/blocks/Covers/CampaignCovers.js

```diff
@@ -0,0 +1,52 @@
+import classnames from 'classnames';
+import { IMAGE_SIZES } from './imageSizes';
+const { __ } = wp.i18n;
+
+export const CampaignCovers = ({ covers, initialRowsLimit, row, loadMoreCovers }) => {
+  const amountPerRow = 3;
+  const showLoadMore = !!initialRowsLimit && covers.length > amountPerRow * row;
+
+  return (
+    <div className='container'>
+      <div className='thumbnail-largeview-container'>
+        {covers.map((cover, index) => {
+          const { href, image, alt_text, name, src_set } = cover;
+          const hideCover = !!initialRowsLimit && index >= row * amountPerRow;
+          return (
+            <div key={name} className={classnames('campaign-card-column', { hidden : hideCover })}>
```

Comment:
It's actually better if we don't render what is not shown. The images for example will probably start loading for all rows.

Reply:
Updated! 🙂